bin/cylc-cat-log

```diff
@@ -296,6 +335,10 @@ def main():
             cmd_tmpl = str(GLOBAL_CFG.get_host_item(
                 "local tail command template"))
             commands.append(shlex.split(cmd_tmpl % {"filename": filename}))
+    elif options.geditor or options.editor:
+        command_list = editor.split()
```

Comment:
@challurip - if you can make this change and push it up today, I'll merge this for the new release (tonight, if the anesthetic has worn off...)